src/change/models/changeManager.ts

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable no-fallthrough */ // the rule is not typescript aware in this case
```

Comment:
please do not ignore the rule and add default cases

Reply:
done.